### ISSUE TITLE
Parse `/addon` directory

### ISSUE THREAD

**Issue by elidupuis** [sivakumar-kailasam/broccoli-leasot#31]
Currently it seems that only the `/app` directory is getting parsed, but other addons might want to use this package too! It would be great if the `/addon` directory was included as well (if it exists!).

**Comment by sivakumar-kailasam** [comment]
Yeah this is something I wanted to look into as well. Will try to ship this sometime this week or the next :smile:

**Comment by elidupuis** [comment]
I poked around a little bit and was going to PR something but I never really wrapped my head around it :wink:

**Comment by sivakumar-kailasam** [comment]
I would most probably look at this over the weekend so if you were planning to send PR before that it'd be awesome.

**Comment by elidupuis** [comment]
I don't suppose there's been any movement on this? Kind of slipped off my radar 😊

**Comment by sivakumar-kailasam** [comment]
Ah no, will get this fixed.
